apps/web/src/styles/globals.css

```
@@ -18,7 +18,18 @@
-  --font-sans: "Plus Jakarta Sans", -apple-system, BlinkMacSystemFont, sans-serif;
+  /* Editorial palette */
+  --color-paper: var(--color-paper);
+  --color-paper-deep: var(--color-paper-deep);
+  --color-ink: var(--color-ink);
+  --color-ink-muted: var(--color-ink-muted);
+  --color-cream: var(--color-cream);
+  --color-cream-muted: var(--color-cream-muted);
+  --color-cream-faint: var(--color-cream-faint);
+  --color-violet-bright: var(--color-violet-bright);
+
+  --font-sans: "Inter Tight", -apple-system, BlinkMacSystemFont, sans-serif;
```

Stylelint (17.7.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)